### ISSUE TITLE
browse_dataset.py in dev branch is extremely slow

### ISSUE THREAD

**Issue by zero0kiriyu** [open-mmlab/mmyolo#679]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmyolo/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmyolo.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmyolo).

### 🐞 Describe the bug

browse_dataset.py in dev branch is extremely slow

https://github.com/open-mmlab/mmyolo/blob/5a96fcdec069b746a4677e17e8cb093c37a4761f/tools/analysis_tools/browse_dataset.py#L177

I think this code wolud make the PackDetInputs deepcopy every key including the dataset, which is used for mosaic and mixup.

### Environment

mmyolo dev branch

**Comment by hhaAndroid** [comment]
@zero0kiriyu Yes. If your image size is large it will be very slow and we will fix it as soon as possible
